docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,104 @@
+Aggregating the readings of a [`GPS`](/components/movement-sensor/gps/) and [`IMU`]((/components/movement-sensor/gps/)) movement sensor in a `merged` model allows you to reduce velocity error when your robot is trying to [navigate](/services/navigation/) with these position and orientation readings.
+{{% /alert %}}
+
+Before configuring a `merged` movement sensor, configure each movement sensor you want to merge as an individual component according to its [model's configuration instructions](/components/movement-sensor/#configuration).
+Reference the `name` you configure for each individual component in the `merged` sensor's configuration attributes as shown:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your movement sensor, select the `movement-sensor` type, and select the `merged` model.
+
+Click **Create Component**.
+
+{{< imgproc src="/components/movement-sensor/merged-config-builder.png" alt="Creation of an `merged` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "merged",
+      "attributes": {
+        "position": ["<your-gps-sensor-name-1>", "<your-gps-sensor-name-2>"],
+        "orientation" : ["<your-imu-sensor-name-1>"],
+        "compass_heading" : ["<your-gps-sensor-name-1>"],
+        "angular_velocity":["<your-angular-velocity-sensor-name-2>"],
+        "linear_velocity": ["<your-gps-sensor-name-1>"],
+        "linear_acceleration": ["<your-adxl345-sensor-name-1>"]
+      },
+    "depends_on": [] 
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// { "components": [ { ... },
+{
+    "name": "<your-sensor-name>",
+    "type": "movement_sensor",
+    "model": "merged",
+    "attributes": {
+        "position": ["gps1"],
+        "orientation" : ["vectornav"],
+        "compass_heading" : ["gps1"],
+        "angular_velocity":[""],
+        "linear_velocity": ["gps1"],
+        "linear_acceleration": ["adxl345"]
+    }, 
+      "depends_on":[]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
+Configure an array of the `name` of each movement sensor you want to add to your robot as a merged resource in the attributes of the `merged` movement sensor model:
+
+- The name of each attribute represents the `Property` that that particular movement sensor supports, or the type of reading or measurement that it takes.
+- Retrieve this information for your movement sensor by calling [`GetProperties()`](/components/movement-sensor/#getproperties) on the sensor.
```

Comment:
```suggestion
- Retrieve this information for your movement sensor from its [model configuration documentation](/components/movement-sensor/#configuration) or by calling [`GetProperties()`](/components/movement-sensor/#getproperties) on the sensor.
```

Realized ppl really shouldn't need to be calling `GetProperties` a lot of the time, we'll just add the properties info to the model's docs right-- maybe this usage info would be more useful with module registry... lmk what you think about changing this part or not! @randhid

Reply:
give the user an either or option. Also mention that we allow you to use the same sensor for multipleattirbutes if it supports multiple properties.

Reply:
Ok, cool! Thank you for the suggestion!